packages/fast-components/src/anchored-region/anchored-region.template.ts

```diff
@@ -0,0 +1,14 @@
+import { html, ref, when } from "@microsoft/fast-element";
+import { AnchoredRegion } from "./anchored-region";
+
+export const AnchoredRegionTemplate = html<AnchoredRegion>`
+  <template>
+      <div
+          class="region"
```

Comment:
Should region be able to be styled externally? If so, it might be worth adding a `part="region"` to it

Reply:
yep